DQMOffline/Configuration/python/DQMOfflineCosmics_cff.py

```diff
@@ -35,7 +36,8 @@
 from DQMOffline.Trigger.DQMOffline_Trigger_cosmics_cff import *
 from DQM.Physics.DQMPhysics_cff import *
 
-DQMOfflineCosmicsPrePOG = cms.Sequence( muonCosmicMonitors *
+DQMOfflineCosmicsPrePOG = cms.Sequence( TrackingDQMTier0 *
```

Comment:
the name TrackingDQMTier0 comes from the original naming scheme in the strip sequence,
which is SiStripDQMTier0